manifest.yml

```diff
@@ -1,11 +0,0 @@
----
-buildpack: go_buildpack
-services:
-  - cdn-creds
-  - cdn-rds
-applications:
-  - name: cdn-broker
-    command: cdn-broker
-  - name: cdn-cron
-    command: cdn-cron
-    no-route: true
```

Comment:
This is because some versions of the cf cli have trouble pushing manifests with multiple applications: https://github.com/cloudfoundry/cli/issues/818. Autopilot doesn't do a good job deploying multiple apps in the same command, but it should be able to deploy a single app from a multi-app manifest correctly as of https://github.com/concourse/autopilot/commit/2ee2da2e36671ce3522079d998393a2ac717dd80.